### PR TITLE
Add remove

### PR DESCRIPTION
**Pull request by josuerushanika**
**To Do list: add & remove**

In this project, l implement the CRUD (create, read, update, delete) methods.

_Project requirements_

Implement a function for adding a new task (add a new element to the array).
Implement a function for deleting a task (remove an element from the array).
Implement a function for editing task descriptions.
By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).
Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1)..
All changes to the To Do List should be saved in local storage.

### CODE REVIEW

**Comment by codecaiine** [comment]
Hi @josuerushanika  I sent you message on slack

File: src/index.js [hunk 1]
```diff
@@ -1,40 +1,120 @@
...
+  }
+  return Taskarray;
+};
+
+// refresh items
+
+const refreshItems = (Taskarray) => {
+  for (let i = 1; i < Taskarray.length; i += 1) {
+    const indexes = i + 1;
+    Taskarray[i].index = indexes;
```
Comment (codecaiine) [inline src/index.js]: - [ ] Good job so far we can add and remove :clap: But when I deleted one task, the task didn't update all remaining items' indexes, and are not representing the current list order and are not unique. Please fix it in order to make your project follow requirements and works as expected :+1: 

#### Current list order
Screenshot_20230127_004939

##### When I deleted one
Screenshot_20230127_005039

File: src/style.css [hunk 2]
```diff
@@ -106,9 +106,19 @@ input:focus {
 }
 
 .todo-footer {
-  background-color: #f7f7f7;
+  background: rgb(209, 123, 123);
   text-align: center;
   padding: 16px;
+  font-weight: 900;
```
Comment (codecaiine) [inline src/style.css:122]: - [ ] [OPTIONAL] I would like to strongly recommend you make sure  Your task is to build a website that displays a list that looks and behaves like the part of the minimalist project captured in the video below. :arrow_down:  to give a good user experience. Please fix the design :+1: 

68747470733a2f2f696d672e796f75747562652e636f6d2f76692f416355642d5f596a6a71672f302e6a7067

##### Your design
Screenshot_20230127_005936(1)

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @josuerushanika , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Great project implementation :heavy_check_mark: 
- No linters errors :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (josuerushanika) [reply src/index.js]: Thx it is fixed

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @josuerushanika 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - No linter errors. ✔️ 
 - Correct use of GitHub workflow. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by josuerushanika** [comment]
Thank you

**Pull request merged**